frontend/src/components/RFI/RFILanding.vue

```diff
@@ -852,152 +799,63 @@
+                    :disabled="isReadOnly"
+                    full-width
+                    max-width="200px"
+                    label="open"
+                  />
 
-                <v-col class="col-md-1 col-12">
-                  <v-menu
-                    ref="menuto"
-                    v-model="obj.menuto"
-                    v-model:return-value="timeto"
+                  <!-- we need logic to prevent choosing a time before -->
+
+                  <AppTimeInput
+                    v-model="obj.timeto"
                     :disabled="isReadOnly"
-                    :close-on-content-click="false"
-                    :nudge-right="40"
-                    transition="scale-transition"
-                    offset-y
-                    max-width="290px"
-                    min-width="290px"
-                  >
-                    <template #activator="{ props }">
-                      <v-text-field v-model="obj.timeto" :disabled="isReadOnly" label="To" readonly v-bind="props" />
-                    </template>
-                    <v-time-picker
-                      v-if="obj.menuto"
-                      v-model="obj.timeto"
-                      :disabled="isReadOnly"
-                      :min="obj.timefrom"
-                      full-width
-                      @click:minute="$refs.menuto[index].save(timeto)"
-                    />
-                  </v-menu>
+                    full-width
+                    max-width="200px"
+                    label="close"
+                  />
                 </v-col>
 
                 <v-col class="col-md-1 col-12">
-                  <v-menu
-                    ref="newmenufrom"
-                    v-model="obj.newmenufrom"
-                    v-model:return-value="newtimefrom"
-                    :close-on-content-click="false"
-                    :nudge-right="40"
-                    transition="scale-transition"
-                    offset-y
-                    max-width="290px"
-                    min-width="290px"
-                  >
-                    <template #activator="{ props }">
-                      <v-text-field
-                        v-model="obj.newtimefrom"
-                        :disabled="isReadOnly"
-                        label="From"
-                        readonly
-                        v-bind="props"
-                      />
-                    </template>
-                    <v-time-picker
-                      v-if="obj.newmenufrom"
-                      v-model="obj.newtimefrom"
-                      full-width
-                      @click:minute="$refs.newmenufrom[index].save(newtimefrom)"
-                    />
-                  </v-menu>
-                </v-col>
+                  <AppTimeInput
+                    v-model="obj.newtimefrom"
+                    :disabled="isReadOnly"
+                    full-width
+                    max-width="200px"
+                    label="open"
```

Comment:
I think the original label was 'From'. Do we have a requirement for changing it to 'open'?

Reply:
done